src/citrine/resources/project.py

```diff
@@ -2,44 +2,46 @@
-from citrine.resources.workflow import WorkflowCollection
-from citrine.resources.dataset import DatasetCollection
-from citrine.resources.condition_template import ConditionTemplateCollection
-from citrine.resources.parameter_template import ParameterTemplateCollection
-from citrine.resources.project_member import ProjectMember
-from citrine.resources.project_roles import MEMBER, ROLES, ACTIONS
-from citrine.resources.property_template import PropertyTemplateCollection
+from citrine.resources.design_workflow import DesignWorkflowCollection
+from citrine.resources.gemtables import GemTableCollection
+from citrine.resources.ingredient_run import IngredientRunCollection
+from citrine.resources.ingredient_spec import IngredientSpecCollection
+from citrine.resources.material_run import MaterialRunCollection
+from citrine.resources.material_spec import MaterialSpecCollection
 from citrine.resources.material_template import MaterialTemplateCollection
+from citrine.resources.measurement_run import MeasurementRunCollection
+from citrine.resources.measurement_spec import MeasurementSpecCollection
 from citrine.resources.measurement_template import MeasurementTemplateCollection
-from citrine.resources.process_template import ProcessTemplateCollection
+from citrine.resources.module import ModuleCollection
+from citrine.resources.parameter_template import ParameterTemplateCollection
+from citrine.resources.predictor import PredictorCollection
+from citrine.resources.predictor_evaluation_execution import \
+    PredictorEvaluationExecutionCollection
+from citrine.resources.predictor_evaluation_workflow import \
+    PredictorEvaluationWorkflowCollection
 from citrine.resources.process_run import ProcessRunCollection
 from citrine.resources.process_spec import ProcessSpecCollection
-from citrine.resources.measurement_run import MeasurementRunCollection
-from citrine.resources.measurement_spec import MeasurementSpecCollection
-from citrine.resources.material_run import MaterialRunCollection
-from citrine.resources.material_spec import MaterialSpecCollection
-from citrine.resources.ingredient_run import IngredientRunCollection
-from citrine.resources.ingredient_spec import IngredientSpecCollection
-
-from citrine._rest.collection import Collection
-from citrine._rest.resource import Resource
-from citrine._serialization import properties
-from citrine.resources.gemtables import GemTableCollection
+from citrine.resources.process_template import ProcessTemplateCollection
+from citrine.resources.processor import ProcessorCollection
+from citrine.resources.project_member import ProjectMember
+from citrine.resources.project_roles import MEMBER, ROLES, ACTIONS
+from citrine.resources.property_template import PropertyTemplateCollection
+from citrine.resources.response import Response
+from citrine.resources.table_config import TableConfigCollection
 from citrine.resources.user import User
+from citrine.resources.workflow import WorkflowCollection
```

Comment:
That's quite a diff.

Reply:
IntelliJ optimize imports. Mistakes were made. Regrets were had.